src/components/checkbox/checkbox.scss

```diff
@@ -105,6 +105,24 @@ $checkbox-top: 12px !default;
     cursor: default;
   }
 
+  &[indeterminate="true"] ._md-icon{
```

Comment:
Can you not apply a custom class to the element (like `.md-indeterminate`)? 

This will be more compatible with the attribute value.